cmd/revad/runtime/runtime.go

```diff
@@ -115,15 +119,49 @@ func New(config *config.Config, opt ...Option) (*Reva, error) {
 		return nil, err
 	}
 
-	return &Reva{
+	r := &Reva{
 		config:     config,
 		servers:    servers,
 		serverless: serverless,
 		watcher:    watcher,
 		lns:        listeners,
 		pidfile:    opts.PidFile,
 		log:        log,
-	}, nil
+	}
+	r.initConfigDumper()
+	return r, nil
+}
+
+func (r *Reva) initConfigDumper() {
+	// dump the config when the process receives a SIGUSR1 signal
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGUSR1)
+
+	go func() {
+		<-sigs
+		r.dumpConfig()
+	}()
+}
+
+func (r *Reva) dumpConfig() {
+	cfg := r.config.Dump()
+	// TODO: merge this config with the one received from
+	// the running services
+	out := r.config.Core.ConfigDumpFile
```

Comment:
Is this system default or hardcoded to `/tmp`?

Reply:
this is configurable in the `core` config:

```
[core]
config_dump_file = "..."
```

the default is `/tmp/reva-dump.toml`

Reply:
@gmgigi96 can you append filename to https://pkg.go.dev/os#TempDir?

Default is `/tmp` but if someones overwrites the env variable is a good practice in UNIX to obey

Reply:
@labkode done!